Provide link to how uptime is managed (#9538)
github/docs: content/codespaces/getting-started/deep-dive.md

### Closing or stopping your codespace

> To stop your codespace you can [use the command palette](/codespaces/codespaces-reference/using-the-command-palette-in-codespaces#suspending-or-stopping-a-codespace) (`Shift + Command + P` (Mac) / `Ctrl + Shift + P` (Windows)). If you exit your codespace without running the stop command, i.e. by closing the browser tab, your browser tab going to sleep, the codespace and its running processes will continue until a window of inactivity occurs, after which the codespace will stop.  

Reviewer: I think the usual user experience is probably to put their machine to sleep, so I updated the wording to match that and make it more relatable. I also mentioned here what the window of inactivity is. Even though it's mentioned about, this just gives the user some more content.

```suggestion
To stop your codespace you can [use the command palette](/codespaces/codespaces-reference/using-the-command-palette-in-codespaces#suspending-or-stopping-a-codespace) (`Shift + Command + P` (Mac) / `Ctrl + Shift + P` (Windows)). If you exit your codespace without running the stop command, i.e. by closing the browser tab or letting your machine go to sleep, the codespace and its running processes will continue until a window of inactivity occurs, after which the codespace will stop.  By default, the window of inactivity is 30 minutes. 
```

Author: Hi Amy, 
Can we simplify this further, in that the browser doesn't need to be closed or machine put in sleep, there is a blanket if there is no activity for 30 minutes your codespace will be stopped. I've noticed that happening when I've been using it. 
Also what happens if using remotely and not in a browser, does the same inactivity window apply.

Reviewer: Yep! The inactivity window applies regardless of method of connection to the Codespace.

> To stop your codespace you can [use the command palette](/codespaces/codespaces-reference/using-the-command-palette-in-codespaces#suspending-or-stopping-a-codespace) (`Shift + Command + P` (Mac) / `Ctrl + Shift + P` (Windows)). If you exit your codespace without running the stop command, i.e. by closing the browser tab or letting your machine go to sleep, the codespace and its running processes will continue until a window of inactivity occurs, after which the codespace will stop.  By default, the window of inactivity is 30 minutes. 

Reviewer: @simonsabin Making a suggestion based on your previous comment. What about:

```suggestion
To stop your codespace you can [use the command palette](/codespaces/codespaces-reference/using-the-command-palette-in-codespaces#suspending-or-stopping-a-codespace) (`Shift + Command + P` (Mac) / `Ctrl + Shift + P` (Windows)). If you exit your codespace without running the stop command (for example, closing the browser tab), or if you leave the codespace running without interaction, the codespace and its running processes will continue until a window of inactivity occurs, after which the codespace will stop.  By default, the window of inactivity is 30 minutes. 
```
